tests/input/example-class.php

```diff
@@ -122,6 +122,7 @@ public function throwWhenInvalid() : void
     public function trySwitchSpace() : void
     {
         try {
+            $var = 1;
```

Comment:
This kinda snuck in: not a problem for me as long as it doesn't remove newlines before control structures, but seems unrelated

Reply:
I just ensured the settings for space before block control structures is not modified from current behaviour. The test for it was missing.